RIPS/rip-9998.md

```diff
@@ -0,0 +1,306 @@
+---
+eip: 0
+title: Aggregation for Native Account Abstraction
+description: Aggregation allows Smart Contract Accounts to share a single signature data between multiple AA transactions.
+author: Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Dror Tirosh (@drortirosh), Shahaf Nacson (@shahafn)
+discussions-to: https://www.google.com/
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-01
+requires: 7560
```

Comment:
If we remove validation/execution separation from 7560 and move it to [PR3](https://github.com/eth-infinitism/RIPs/pull/3/files) than this RIP probably also requires that.